### PR TITLE
Users/shdeb/fix release

### PR DESCRIPTION
**Pull request by Shuvajyoti**
Creating a PR to fix sign and publish scripts.
Currently there is an issue for adding Kv N/W rule where the az cli command for this fails with error incorrect API version. A fix for this is provided as adding the resource group parameter into the az cli command (refer - https://github.com/Azure/azure-cli/issues/27147).
Added this parameter and the release is working as expected. Consumers need to add the new secret (KEY_VAULT_RESOURCE_GROUP) in their consuming pipeline.
Also, included earlier fixes for publish script.

### CODE REVIEW

**Review by ms-mikeb — APPROVED** [review]
You may need to remove @jarz from the CODEOWNERS if we can't get ahold of him for an approval.

**Review by agrawalamit31 — APPROVED** [review]

**Pull request merged**
